spec/latest/core/index.html

```diff
@@ -944,8 +944,8 @@ <h4>"Manages block"</h4>
       -->
     </pre>
 
-    <p>A manages block MUST use two and only two of the <code>subject</code>, <code>property</code>
-      and <code>object</code> predicates. There MAY be more than one such manages blocks,
+    <p>A <i>memberAssertion</i> block MUST use two and only two of the <code>subject</code>, <code>property</code>
+      and <code>object</code> predicates. There MAY be more than one such blocks,
```

Comment:
```suggestion
    <p>A <i>memberAssertion</i> MUST use two and only two of the <code>subject</code>, <code>property</code>
      and <code>object</code> predicates. The <code>memberAssertion</code> property MAY have more than one object,
```

Reply:
I've replaced _property_MAY_ with _predicate_MAY_ - I didn't want to use that word having in mind the `hydra:property`